src/Components/HeroImageOfTheDay/HeroImageOfTheDay.scss

```diff
@@ -0,0 +1,6 @@
+﻿//Please read README section on Styling before editing or creating SCSS
+@import "./src/ConstantsMixins";
+
+.imageOfTheDay {
+  @include hero-astronomy-image-of-the-day();
```

Comment:
I think this is only used here, so its probably easiest for it to not be a mixin (ie, just put the code here)
Mixins are great for when we want to re-use things all over a project.